core/ir/disassemble_shared.c

```diff
@@ -809,6 +817,48 @@ print_extra_bytes_to_file(file_t outfile, byte *pc, byte *next_pc, int extra_sz,
     os_write(outfile, buf, sofar);
 }
 
+/* DR_ISA_REGDEPS instruction encodings can be at most 16 bytes,
+ * hence we can have at most 2 lines.
+ */
+#define REGDEPS_BYTES_PER_LINE 8
+
+int
```

Comment:
static (don't make global functions without a good reason)

Reply:
Moved to core/ir/isa_regdeps/disassemble.c.